ethereum/contracts/pyth/Pyth.sol

```diff
@@ -218,7 +218,8 @@ abstract contract Pyth is PythGetters, PythSetters, AbstractPyth {
 
         // Check that there is not a significant difference between this chain's time
         // and the price publish time.
-        if (diff(block.timestamp, info.priceFeed.publishTime) > validTimePeriodSeconds()) {
+        if (info.priceFeed.status == PythStructs.PriceStatus.TRADING && 
```

Comment:
is this trading check right? what if we add a new status flag in the future -- do we not want to change that status to unknown?

I think dropping this check is probably better.

Reply:
I think when we add a new status we will probably handle all cases at that time 😅